### PR TITLE
fix(Pocket Casts): make presence display page and audio listening to

### LINKED ISSUE #2696: Pocket Casts | play.pocketcasts.com
**Describe the Bug**

When you launch a podcast on Pocket Casts, the presence just says "viewing subscriptions" no matter what.

**Steps to Reproduce**

1. Go on https://play.pocketcasts.com.
2. Click play on a podcast.
3. Watch as nothing changes in the presence on Discord.

**Expected Behavior**

I'm expecting it to show the episode name of whatever I'm listening to, which podcast it's from and maybe the time stamp.

**Screenshots**

Bug in action

**Environment**

- OS:  Windows 10 20H2
- Browser:  Firefox (but I think it happens on every browser)
- Version:  84.0.1 (64-bit)
  

**Additional context**

This is a major bug that makes the presence unusable.

### PR DESCRIPTION
**Pull request by realtofuine**
## Description 
make Pocket Casts Rich Presence work again

fixes #2696 

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image

image

image

image

</details>

### CODE REVIEW

**Comment by realtofuine** [comment]
fixes #2696

**Comment by ImRodry** [comment]
Please use a proper title (say what was actually wring and what you fixed)

File: websites/P/Pocket Casts/presence.ts [hunk 1]
```diff
@@ -7,9 +7,39 @@ presence.on("UpdateData", async () => {
 		largeImageKey: "icon",
 	};
 
-	if (document.querySelectorAll(".player-controls").length === 0) {
-		presenceData.smallImageKey = "more";
+	// const el = document.querySelector(".controls");
```
Comment (ImRodry) [inline websites/P/Pocket Casts/presence.ts:10]: why

File: websites/P/Pocket Casts/presence.ts [hunk 2]
```diff
@@ -7,9 +7,39 @@ presence.on("UpdateData", async () => {
...
+		presenceData.state = document.querySelector(
+			"div.podcast-title-date > span"
+		).textContent;
+
+		const time = document
+			.querySelector(
+				".TimeTextstyled__TimeTextWrapper-sc-1yzkn0m-0.lmVDeu.current-time"
+			)
+			.textContent.split(":")
+			.map(n => Number(n));
```
Comment (ImRodry) [inline websites/P/Pocket Casts/presence.ts:24]:
```suggestion
			.map(Number);
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/P/Pocket Casts/presence.ts (hunk 1 above)
Comment (realtofuine) [reply websites/P/Pocket Casts/presence.ts:10]: oops lol
i was testing

**Review by ImRodry — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Thanks for the PR

**Pull request merged**